nf_core/lint/multiqc_config.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import os
+import yaml
+
+
+def multiqc_config(self):
+    """Make sure basic multiQC plugins are installed and plots are exported
+    Basic template:
+    report_comment: >
+        This report has been generated by the <a href="https://github.com/nf-core/quantms" target="_blank">nf-core/quantms</a>
+        analysis pipeline. For information about how to interpret these results, please see the
+        <a href="https://nf-co.re/quantms" target="_blank">documentation</a>.
+    report_section_order:
+        software_versions:
+            order: -1000
+        nf-core-quantms-summary:
+            order: -1001
+
+    export_plots: true
+    """
+    passed = []
+    failed = []
+
+    fn = os.path.join(self.wf_path, "assets", "multiqc_config.yaml")
+
+    # Return a failed status if we can't find the file
+    if not os.path.isfile(fn):
+        return {"failed": ["'assets/multiqc_config.yaml' not found"]}
+
+    try:
+        with open(fn, "r") as fh:
+            mqc_yml = yaml.safe_load(fh)
+    except Exception as e:
+        return {"failed": ["Could not parse yaml file: {}, {}".format(fn, e)]}
+
+    # Check that the report_comment exists and matches
+    try:
+        assert "report_section_order" in mqc_yml
+        orders = dict()
+        summary_plugin_name = "nf-core-" + self.pipeline_name + "-summary"
+        min_plugins = ["software_versions", summary_plugin_name]
+        for plugin in min_plugins:
+            assert plugin in mqc_yml["report_section_order"]
+            assert "order" in mqc_yml["report_section_order"][plugin]
+            plugin_order = mqc_yml["report_section_order"][plugin]["order"]
+            assert plugin_order < 0
```

Comment:
Might be nice to attach explanations to these exceptions, and then include them in the failed message. For example:
```suggestion
            assert plugin_order < 0, f"Section {plugin} 'order' must be < 0"
```

Reply:
Is it enough in the asserts or should I collect strings to create a message that contains everything that went wrong?

Reply:
Ah see your next comment.